### PR TITLE
Update python samples

### PR DESCRIPTION
**Pull request by sturkmen72**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [x] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: samples/python/facedetect.py [hunk 1]
```diff
@@ -39,13 +39,13 @@ def main():
     except:
         video_src = 0
     args = dict(args)
-    cascade_fn = args.get('--cascade', "data/haarcascades/haarcascade_frontalface_alt.xml")
-    nested_fn  = args.get('--nested-cascade', "data/haarcascades/haarcascade_eye.xml")
+    cascade_fn = args.get('--cascade', "../../data/haarcascades/haarcascade_frontalface_alt.xml")
+    nested_fn  = args.get('--nested-cascade', "../../data/haarcascades/haarcascade_eye.xml")
```
Comment (alalek) [inline samples/python/facedetect.py:43]: > ../../data

Should not be used.
`cv.samples.findFile()` is applied below.
We may remove `data`

Ref: https://github.com/opencv/opencv/blob/4.5.5/modules/core/src/utils/samples.cpp#L33-L35

**Review by alalek — COMMENTED** [review]
Thank you for contribution!

File: samples/python/facedetect.py (hunk 1 above)
Comment (sturkmen72) [reply samples/python/facedetect.py:43]: @alalek
```
    cascade_fn = args.get('--cascade', "haarcascades/haarcascade_frontalface_alt.xml")
    nested_fn  = args.get('--nested-cascade', "haarcascades/haarcascade_eye.xml")
```
this doesn't work for me.

works when i copy haarcascades directory `opencv\data\haarcascades` -> `opencv\samples\data\haarcascades`

Comment (alalek) [reply samples/python/facedetect.py:43]: Please dump log messages with `OPENCV_LOG_LEVEL=DEBUG` environment variable.

Comment (sturkmen72) [reply samples/python/facedetect.py:43]:
```
[DEBUG:0@0.016] global D:\GitHub\opencv\modules\core\src\utils\samples.cpp (54) cv::samples::findFile cv::samples::findFile('haarcascades/haarcascade_eye_tree_eyeglasses.xml', false)
[DEBUG:0@0.016] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (197) cv::utils::findDataFile utils::findDataFile('haarcascades/haarcascade_eye_tree_eyeglasses.xml', OPENCV_SAMPLES_DATA_PATH)
[DEBUG:0@0.017] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (212) cv::utils::findDataFile ... Line 212: trying open 'haarcascades/haarcascade_eye_tree_eyeglasses.xml'
[DEBUG:0@0.018] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (262) cv::utils::findDataFile utils::findDataFile(): trying OPENCV_SAMPLES_DATA_PATH=D:\GitHub\opencv\samples\data
[DEBUG:0@0.018] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (267) cv::utils::findDataFile ... Line 267: trying open 'D:\GitHub\opencv\samples\data\haarcascades/haarcascade_eye_tree_eyeglasses.xml'
[DEBUG:0@0.018] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (267) cv::utils::findDataFile ... Line 267: trying open 'D:\GitHub\opencv\samples\data\data\haarcascades/haarcascade_eye_tree_eyeglasses.xml'
[DEBUG:0@0.019] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (267) cv::utils::findDataFile ... Line 267: trying open 'D:\GitHub\opencv\samples\data\samples/data\haarcascades/haarcascade_eye_tree_eyeglasses.xml'
[ INFO:0@0.019] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (277) cv::utils::findDataFile utils::findDataFile(): can't find data file via OPENCV_SAMPLES_DATA_PATH configuration override: haarcascades/haarcascade_eye_tree_eyeglasses.xml
[ WARN:0@0.019] global D:\GitHub\opencv\modules\core\src\utils\samples.cpp (61) cv::samples::findFile cv::samples::findFile('haarcascades/haarcascade_eye_tree_eyeglasses.xml') => ''
WARNING: Could not load classifier cascade for nested objects
[DEBUG:0@0.024] global D:\GitHub\opencv\modules\core\src\utils\samples.cpp (54) cv::samples::findFile cv::samples::findFile('haarcascades/haarcascade_frontalface_alt.xml', true)
[DEBUG:0@0.024] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (197) cv::utils::findDataFile utils::findDataFile('haarcascades/haarcascade_frontalface_alt.xml', OPENCV_SAMPLES_DATA_PATH)
[DEBUG:0@0.024] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (212) cv::utils::findDataFile ... Line 212: trying open 'haarcascades/haarcascade_frontalface_alt.xml'
[DEBUG:0@0.024] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (262) cv::utils::findDataFile utils::findDataFile(): trying OPENCV_SAMPLES_DATA_PATH=D:\GitHub\opencv\samples\data
[DEBUG:0@0.025] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (267) cv::utils::findDataFile ... Line 267: trying open 'D:\GitHub\opencv\samples\data\haarcascades/haarcascade_frontalface_alt.xml'
[DEBUG:0@0.026] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (267) cv::utils::findDataFile ... Line 267: trying open 'D:\GitHub\opencv\samples\data\data\haarcascades/haarcascade_frontalface_alt.xml'
[DEBUG:0@0.027] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (267) cv::utils::findDataFile ... Line 267: trying open 'D:\GitHub\opencv\samples\data\samples/data\haarcascades/haarcascade_frontalface_alt.xml'
[ INFO:0@0.027] global D:\GitHub\opencv\modules\core\src\utils\datafile.cpp (277) cv::utils::findDataFile utils::findDataFile(): can't find data file via OPENCV_SAMPLES_DATA_PATH configuration override: haarcascades/haarcascade_frontalface_alt.xml
[ WARN:0@0.028] global D:\GitHub\opencv\modules\core\src\utils\samples.cpp (61) cv::samples::findFile cv::samples::findFile('haarcascades/haarcascade_frontalface_alt.xml') => ''
OpenCV: terminate handler is called! The last OpenCV error is:
OpenCV(4.5.5-dev) Error: Unspecified error (OpenCV samples: Can't find required data file: haarcascades/haarcascade_frontalface_alt.xml) in cv::samples::findFile, file D:\GitHub\opencv\modules\core\src\utils\samples.cpp, line 64
```

**Review by alalek — APPROVED** [review]
Thank you for contribution :+1:

**Pull request merged**